api/v1/lib/resources.go

```diff
@@ -226,6 +226,23 @@ func (resources Resources) String() string {
 		buf.WriteString(")")
 		if d := r.GetDisk(); d != nil {
 			buf.WriteString("[")
+			if s := d.GetSource(); s != nil {
+				switch s.GetType() {
+				case PATH:
+					buf.WriteString("(PATH:")
+					if p := s.GetPath(); p != nil {
+						buf.WriteString(p.GetRoot())
+					}
+				case MOUNT:
+					buf.WriteString("(MOUNT:")
+					if m := s.GetMount(); m != nil {
+						buf.WriteString(m.GetRoot())
+					}
+				default:
+					buf.WriteString("(ROOT:")
```

Comment:
can we get rid of `ROOT:` output here? I'd like for the output to be mostly consistent with https://github.com/apache/mesos/blob/1.2.x/src/common/resources.cpp#L1922